docs/blog/08-21-2024-introducing-the-federation-api.md

```diff
@@ -0,0 +1,402 @@
+available for files that are not _yet_ available, but will be made available at
+some point in the future).
+
+`Gateway`s wrap `Link`s, which describe how the data can be accessed once you've
+fulfilled the authentication and authorization requirements for that data. There
+are a variety of different avenues for data access supported by the
+specification, including:
+
+- **`Direct` links**, which describe an exact URL that may be accessed. This is
+  useful is the most straightforward case where the data is available by
+  visiting the provided URL.
+- **`Approximate` links**, which provide a URL and then subsequent instructions
+  to follow to access the data. This is useful for when (a) an exact link does
+  not exist (e.g., "go to this page and use the filters on the right") and (b)
+  when describing how to access data that cannot be made available through a
+  single link (e.g., requesting all of the data for a particular sample).
+- **`Informational` links**, which point to an external webpage where you can
+  learn more about how to access the data. This is useful when the source server
+  itself does not have direct knowledge of how to access the data, but it can
+  direct you to a URL where you can learn more.
+- **`MailTo` links**, which support listing an email address that you can
+  contact about accessing the data.
+
+That being said, this list may not be all inclusive today, and growing the list
+of supported link types is an area of future discussion.
+
+Together, the `Gateway` concept (covering _how_ to access the data) combined
+with the `Link` concept (covering _where_ to access the data) provides an
+expressive vocabulary for articulating how data may be accessed within the
+ecosystem. Our goal is to improve the user experience of accessing data shared
+by various federation members and to make it easier to build automated data
+access pipelines that seamlessly download the latest pediatric cancer research
+data as it becomes available. Several examples of how to assign and interpret
+these `Gateway`s/`Link`s are available [in the corresponding documentation]
+(**Clay to add link here**).
+
+### Limitations
+
+Being the first version of the API specification, there are several important
+notes to understand when leveraging the API regarding its limitations. We expect
+the list of notes to grow as we receive feedback and questions on this blog
+post.
+
+#### Interpretation of Samples
+
+As discussed in the description of `Sample`s above, this entity covers
+essentially everything between a `Subject` and a `File`. This means that you
+must take great care when interpreting the results from any `/sample` endpoints.
+For example,
+
+- When a single case has been profiled by whole-genome, whole-exome, and
+  RNA-Seq, the server is expected to return three sample entries—not one.
+  Importantly though, the identifier for the sample should be made unique
+  between the three entries when also considering the experiment in the
```

Comment:
My comment from the previous draft remains: The sample identifier element doesn't have any `experiment` field so I don't understand what this refers to.
Do you mean something to the effect of "these three experiments on the same case will each be assigned a separate sample identifier"?  That's how Treehouse would do it, though the experiment type wouldn't be included in the identifier -- you'd need to use the `library_strategy` metadata to understand what was different about each sample.

Reply:
Ah right—apologies, I did _think_ of response here, but I neglected to update it. I'll change to remove this reference and use `library_strategy`.